eclipsePlugin/build.gradle

```diff
@@ -34,6 +36,12 @@ def eclipsePluginId = 'com.github.spotbugs.plugin.eclipse'
 def localProps = new Properties()
 localProps.load(new FileInputStream("$projectDir/local.properties"))
 
+ext {
+  eclipseRootDir = localProps.getProperty('eclipseRoot.dir')
+  eclipseExecutable = eclipseRootDir + (Os.isFamily(Os.FAMILY_MAC) ? "/Contents/MacOS/eclipse" : "/eclipse")
+  eclipsePluginsDir = eclipseRootDir + (Os.isFamily(Os.FAMILY_MAC) ? "/Contents/Eclipse/plugins" : "/plugins")
+}
```

Comment:
Current stable release is Neon, and Mars isn't supported for now according to [Wikipedia](https://en.wikipedia.org/wiki/Eclipse_(software)#Releases) (sorry I cannot find official page).

Then why we do not drop support for Kepler, Luna and Mars?
It is also necessary to update our `.travis.yml` to use Neon to build the plugin.